fetch.bs

```diff
@@ -2146,14 +2154,15 @@ run these steps:
 </ol>
 
 
-<h3 id=client-hints-list>Client hints list</h3>
+<h3 id=client-hints-list>Client-hints set</h3>
 
-<p class=note>This section will be integrated into HTTP Client Hints.
+<p class=XXX>HTTP Client-Hints requires further integration. More details on the
+<a href="https://github.com/whatwg/fetch/issues/726">issue</a>.
 [[!CLIENT-HINTS]]
 <!-- XXX -->
 
-<p>A <dfn export id=concept-client-hints-list for=client>client hints list</dfn> is a
-<a for=/>list</a> of
+<p>A <dfn export id=concept-client-hints-list>client-hints set</dfn> is a
```

Comment:
Try this?

```suggestion
<p>A <dfn export id=concept-client-hints-list for=/>client-hints set</dfn> is a
```

Reply:
Travis still [doesn't like me](https://travis-ci.org/whatwg/fetch/builds/481359853?utm_source=github_status&utm_medium=notification)